vector/src/appfdroid/java/im/vector/push/fcm/NotificationTroubleshootTestManagerFactory.kt

```diff
@@ -38,7 +40,15 @@ class NotificationTroubleshootTestManagerFactory {
             // mgr.addTest(TestServiceRestart(fragment))
             mgr.addTest(TestAutoStartBoot(fragment))
             mgr.addTest(TestBackgroundRestrictions(fragment))
-            // mgr.addTest(TestBatteryOptimization(fragment))
+
+            // Check whether we're using Fdroid mode
+            val pushManager = Matrix.getInstance(fragment.context).pushManager
+            if (!pushManager.useFcm() || !pushManager.hasRegistrationToken()) {
+                // We're using Fdroid: test the Battery Optimization for RealTime sync mode
```

Comment:
This class is in `appfdroid/` path, so it is for the F-Droid version. So maybe those tests above are redundant?

Reply:
(There is another `NotificationTroubleshootTestManagerFactory` class for Gplay variant)

Reply:
agree - I will update